CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * This library requires Turf v2.0.0-alpha.3. ([#525](https://github.com/mapbox/mapbox-directions-swift/pull/525))
 * The `Incident.impact` property is now an `Incident.Impact` value instead of a string. ([#519](https://github.com/mapbox/mapbox-directions-swift/pull/519))
 
+## v2.0.0
```

Comment:
Even the whole section above is for v2.0.0, but we can clean it up after merging the PR.